ArjunaJTS/narayana-jts-idlj/pom.xml

```diff
@@ -24,11 +24,6 @@
   <name>Narayana: ArjunaJTS narayana-jts-idlj</name>
   <description>JTS uber jar with idlj stubs</description>
   <dependencies>
-    <dependency>
-      <groupId>org.apache.ant</groupId>
-      <artifactId>ant</artifactId>
-      <scope>provided</scope>
-    </dependency>
```

Comment:
I guess this is not used, isn't it?

Reply:
Yes, exactly. It is a test dependency and it is not used there. So instead using the narayana-bom-test I thought it would be better to remove that test dep.